bin/main.ml

```diff
@@ -150,7 +183,8 @@ let draw ui_terminal matrix (game_state : game_state) =
     (* draw letters box *)
     (let ctx = with_grid_cell ctx layout_spec 1 2 0 1 in
      let _ = with_frame ctx " letters " LTerm_draw.Heavy in
-     ());
+     ();
+     draw_letters ctx (List.hd game_state.player_list).letters);
```

Comment:
probably move up this call to `List.hd game_state.player_list` into a let statement defining a variable `current_player` around line 173, since we're going to need the current player to draw both the letter box and the players box